scene/3d/area.cpp

```diff
@@ -750,7 +750,7 @@ Area::Area() :
 	gravity_is_point = false;
 	gravity_distance_scale = 0;
 	linear_damp = 0.1;
-	angular_damp = 1;
```

Comment:
`angular_damp` is initialized to `1` in Area, Area2D, AreaSW and Area2DSW, so if we're changing the default, it should be made consistently.

Reply:
Reading #26416, it seems like only AreaSW should also be changed, as for 2D `1` is the expected default.

Reply:
The global defaults are defined here:
https://github.com/godotengine/godot/blob/29a1202d39b0efb617a8ff4ee9764612f3e0124f/scene/resources/world_2d.cpp#L397
And here:
https://github.com/godotengine/godot/blob/29a1202d39b0efb617a8ff4ee9764612f3e0124f/scene/resources/world.cpp#L335

The issue was about having Area/2Ds match the defaults (I also tried to make them match project overrides, but sadly this only updated when restarting project).

Anyways, I didn't touch AreaSW, because I don't even know what it is. But if this change is required too, I could make an update.

Reply:
Ok, done.